blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractPipelineCreateRequest.java

```diff
@@ -52,6 +55,16 @@ public AbstractPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
     }
 
     protected ModifiableTopLevelItemGroup getParent() {
-        return Jenkins.getInstance();
+        String organization = getOrganization();
+        ItemGroup parent =  OrganizationResolver.getItemGroup(getOrganization());
+        if(parent == null){
+            throw new ServiceException.BadRequestExpception("Invalid Jenkins organization. " + organization + ". Not found");
+        }
+
+        if(!(parent instanceof ModifiableTopLevelItemGroup)){
```

Comment:
@vivek yes, I think we should be doing a blanket `s/ItemGroup/ModifiableItemGroup/` change across the code base here.

In Blue Steel Organizations are defined by a "Team Folder" type which inherits from `Folder` and is `ModifiableItemGroup`. Standalone is the same as `Jenkins` is also `ModifiableItemGroup`.